doc/examples/plot_entropy2.py

```diff
@@ -0,0 +1,38 @@
+"""
+=======
+Entropy
+=======
+
+The following example applies the local entropy measure to a noised image with a variable noise.
+
+"""
+import matplotlib.pyplot as plt
+import numpy as np
+
+from skimage.filters.rank import entropy
+from skimage.morphology import disk
+
+noise_mask = 28*np.ones((128,128),dtype=np.uint8)
```

Comment:
Please use PEP8 and add whitespaces after commas https://www.python.org/dev/peps/pep-0008/